packer/sagebio/rStudioAmi-LATEST/template.json

```diff
@@ -0,0 +1,62 @@
+{
+  "builders": [
+    {
+      "ami_name": "{{user `ImageName`}}",
+      "ami_users": "{{user `AmiUsers`}}",
+      "force_delete_snapshot": "true",
+      "instance_type": "{{user `InstanceType`}}",
+      "launch_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "/dev/xvda",
+          "volume_size": "{{user `VolumeSize`}}",
+          "volume_type": "gp2"
+        }
+      ],
+      "profile": "{{user `AwsProfile`}}",
+      "region": "{{user `AwsRegion`}}",
+      "snapshot_users": "{{user `SnapshotUsers`}}",
+      "source_ami": "{{user `SourceImage`}}",
+      "ssh_username": "{{user `SshUsername`}}",
+      "tags": {
+        "BuiltBy": "Packer",
+        "Department": "{{user `Department`}}",
+        "Name": "{{user `ImageName`}}",
+        "OwnerEmail": "{{user `OwnerEmail`}}",
+        "Project": "{{user `Project`}}",
+        "SourceAmiId": "{{ .SourceAMI }}",
+        "SourceAmiName": "{{ .SourceAMIName }}"
+      },
+      "type": "amazon-ebs"
+    }
+  ],
+  "post-processors": [
+    {
+      "output": "manifest.json",
+      "strip_path": true,
+      "type": "manifest"
+    }
+  ],
+  "provisioners": [
+    {
+      "ansible_env_vars": [
+        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'",
+        "JcConnectKey={{user `JcConnectKey`}}"
```

Comment:
this is no longer used.  hmm, we should test wether new instances are connected to jumpcloud without this.

Reply:
I can open a sandbox provisioning PR